coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,21 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
```

Comment:
It is better to use `original_line=re.compile()`

Reply:
Yes, it is but only when you need to create an object with the required pattern which you have to use further in your code. As here I am using the pattern only once, so it's better to use it without creating object. Bit of memory will be saved!

Reply:
@newbazz but I have to get the entire string. Don't need a regex to do that.